pages/api/cms/market-metadata/[marketid].ts

```diff
@@ -0,0 +1,29 @@
+import { getCmsMarketMetadataForMarket } from "lib/cms/get-market-metadata";
+import { NextApiRequest, NextApiResponse, PageConfig } from "next";
+
+export const config: PageConfig = {
```

Comment:
Did you verify that this works? I think the function signature is a little different for edge: https://vercel.com/docs/functions/edge-functions/edge-functions-api#edge-functions-signature

Would also be good to change the other serverless function in this pr too, probably that one is more important too